src/main/java/baseball/controller/BaseballGameController.java

```diff
@@ -0,0 +1,112 @@
+package baseball.controller;
+
+import baseball.domain.Computer;
+import baseball.domain.User;
+import baseball.util.Validation;
+import camp.nextstep.edu.missionutils.Console;
+
+public class BaseballGameController {
+
+    private int ball;
+    private int strike;
+    private Computer computer;
+    private User user;
+
+    public BaseballGameController() {
+        this.computer = new Computer();
+        this.computer.makeNumber();
+    }
+
+    public void initGame() {
+        this.ball = 0;
+        this.strike = 0;
+        this.user = new User();
+    }
+
+    public boolean startGame() {
+        System.out.println("숫자 야구 게임을 시작합니다.");
+
+        while (true) {
+            initGame();
+            inputNumber();
+            checkNumber();
+            printResult();
+
+            boolean isExit = exitGame();
+            if (isExit) {
+                return isExit;
+            }
+        }
```

Comment:
```suggestion
        do {
            initGame();
            inputNumber();
            checkNumber();
            printResult();

            boolean isExit = exitGame();
        } while (isExit)
        return true;
```

1. 더 적합한 구조를 추천합니다.
2. 더 크게 보면 무조건 true가 반환된다면 true는 불용입니다. 반환값 없이 스스로 동작하는 구조로 바꿀 수 있습니다. main 메서드의 구조를 개선해보시면 좋겠습니다.

Reply:
와 저도 이번 코드리뷰를 통해 알고만있던 do~while문을 사용해볼 생각은 못했습니다.
감사합니다!!